docs/assembly.rst

```diff
@@ -136,6 +136,11 @@ evaluate to the address of the variable in calldata, not the value itself.
 The variable can also be assigned a new offset, but note that no validation to ensure that
 the variable will not point beyond ``calldatasize()`` is performed.
 
+For external function pointers the address and the function selector can be
+accessed using ``x.address`` and ``x.selector``.
+The selector will be left aligned and of type ``uint32``.
```

Comment:
If there's no unused space then isn't it technically both left- and right-aligned? Or not aligned at all? I'm not sure if it makes sense to speak of alignment of a 4-byte value in a 4-byte type.

Reply:
Ah, it probably means alignment in the 32-byte slot. I think it should be clarified here. But then I do not see in what sense it's a `uint32` value - the type depends on the type of an underlying variable.

Reply:
There are no types in assembly, so I would propose to talk about four bytes that are left-aligned or something like that.